src/gulpfile.js

```diff
@@ -5,7 +5,7 @@ var gulp = require('gulp');
 var shell = require('gulp-shell');
 var $ = require('gulp-load-plugins')();
 
-gulp.task('collectstatic', shell.task(['python /vagrant/python/cac_tripplanner/manage.py collectstatic --noinput']));
+gulp.task('collectstatic', shell.task(['python ../python/cac_tripplanner/manage.py collectstatic --noinput']));
```

Comment:
I think this is okay for now, but we may eventually want to revisit this since it requires the `src` and `python` directories to be siblings.